.github/workflows/update_dependencies_zulip.yml

```
@@ -30,7 +30,10 @@ jobs:
           output="$(./scripts/parse_lake_manifest_changes.py)"
           echo "Dependency changes:"
           echo "$output"
-          echo "changes=$output" >> $GITHUB_OUTPUT
+          delimiter="EOF"
+          echo "changes<<$delimiter" >> $GITHUB_OUTPUT
+          echo "$output" >> $GITHUB_OUTPUT
+          echo "$delimiter" >> $GITHUB_OUTPUT
```

How about
```suggestion
          delimiter="EOF"
          printf 'changes<<%s\n%s\n%s' "$delimiter" "${message}" "$delimiter"
```
that I adapted from `.github/workflows/maintainer_merge.yml`?

I'm a bit skeptical about not passing all the lines at once...